src/main.zig

```diff
@@ -242,18 +242,18 @@ fn fillRectMvp(t: *Tetris, color: *const Vec4, mvp: *const Mat4x4) void {
 
 fn fillRect(t: *Tetris, color: *const Vec4, x: f32, y: f32, w: f32, h: f32) void {
     const model = mat4x4_identity.translate(x, y, 0.0).scale(w, h, 0.0);
-    const mvp = t.projection.mult(model);
-    fillRectMvp(t, color, mvp);
+    const mvp = t.projection.mult(&model);
```

Comment:
Instead of this, the better thing to do is change the prototype of the `mult` function to accept a "direct parameter" rather than a const pointer. https://ziglang.org/download/0.3.0/release-notes.html#parameter-passing